app/controllers/openid_connect/logout_controller.rb

```diff
@@ -34,7 +34,7 @@ def show
     # @see {OpenID Connect RP-Initiated Logout 1.0 Specification}[https://openid.net/specs/openid-connect-rpinitiated-1_0.html#RPLogout] # rubocop:disable Layout/LineLength
     def create
       session[:original_method] = request.method.to_s
-      redirect_to action: :show, **logout_params
+      redirect_to action: :show, status: :see_other, **logout_params
```

Comment:
1. Can we add a regression spec that verified the correct redirect status is sent?

2. Is this a correctness/semantics change, or do we expect browsers to behave differently with this different response code?

   - And if this is supposed to correspond to different browser behavior, is there an acceptance spec we can write that would capture that?

Reply:
Browsers do behave differently when testing the IdP app locally vs in the production/int/dev environments (i.e., the idp app must be served behind our load balancer and proxy setup), so I'm not sure what use a regression spec would be here.

Reply:
(According to the RFC, we should have been returning a different status as a matter of correctness anyway.)

Reply:
ok! but we can at least assert the status in the controller specs

```ruby
expect(response).to redirect_to(action: :show, status: :see_other)
```

https://www.rubydoc.info/gems/rspec-rails/RSpec/Rails/Matchers/RedirectTo#redirect_to-instance_method


https://blog.saeloun.com/2022/11/08/assert-redirected-to-http-status-code/#after

Reply:
Is there a way to reproduce the caching issue?

Reply:
@zachmargolis - IMO, a test to determine if the right status is returned (for this particular fix) is actually testing whether Rails consistently sends back the right status code when redirecting actions. That type of test should exist upstream already in the Rails project, so we *shouldn't* need it in our codebase.

@mitchellhenke - I'm going to try and repro this in my personal AWS sandbox today or tomorrow. If I'm not able to do so, I've confirmed with team Melba that it's beyond the scope of this particular work item to get that type of testing apparatus implemented, but I would be willing to help make sure that work gets does get logged (with which team would is TBD).

Reply:
> @zachmargolis - IMO, a test to determine if the right status is returned (for this particular fix) is actually testing whether Rails consistently sends back the right status code when redirecting actions. That type of test should exist upstream already in the Rails project, so we _shouldn't_ need it in our codebase.

Sorry if it wasn't clear, I was asking us to add a regression suite. Not to check that Rails returns the right conversion to 303, but that we are sending the status code we intend to. To add a check against accidental future regressions so we capture the intent in our specs

Reply:
@lmgeorge I'm mostly interested in confirming it, I'm having trouble understanding how the status code would lead to skipping confirmation (and potentially whether something else is going on).

Reply:
Figured out the issue (thank you @mitchellhenke!) and will be closing this PR. See closing comment for more information.